mon-pix/tests/acceptance/challenge-qcm-test.js

```diff
@@ -1,93 +1,158 @@
+      // when
+      await visit(`/assessments/${assessment.id}/challenges/${qcmChallenge.id}`);
+    });
+
+    it('should render challenge information and question', () => {
+      // then
+      expect(find('.challenge-statement__instruction').textContent.trim()).to.equal(qcmChallenge.instruction);
+
+      expect(findAll('input[type="checkbox"]')).to.have.lengthOf(4);
+      expect(findAll('.proposal-text')[0].textContent.trim()).to.equal('possibilite 1, et/ou');
+      expect(findAll('.proposal-text')[1].textContent.trim()).to.equal('possibilite 2, et/ou');
+      expect(findAll('.proposal-text')[2].textContent.trim()).to.equal('possibilite 3, et/ou');
+      expect(findAll('.proposal-text')[3].textContent.trim()).to.equal('possibilite 4');
+
+      expect(find('.alert')).to.not.exist;
+
+    });
+
+    it('should display the alert box if user validates without checking a checkbox', async () => {
+      // when
+      await click('.challenge-actions__action-validate');
+
+      // then
+      expect(find('.alert')).to.exist;
+      expect(find('.alert').textContent.trim()).to.equal('Pour valider, sélectionner au moins une réponse. Sinon, passer.');
+    });
+
+    it('should hide the alert error after the user interact with checkboxes', async () => {
+      // given
+      await click('.challenge-actions__action-validate');
+
+      // when
+      await click(findAll('.proposal-text')[1]);
+
+      // then
+      expect(find('.alert')).to.not.exist;
+    });
+
+    it('should go to checkpoint when user validated', async () => {
+      // when
+      await click(findAll('.proposal-text')[1]);
+      await click('.challenge-actions__action-validate');
+
+      // then
+      expect(currentURL()).to.contains(`/assessments/${assessment.id}/checkpoint`);
+    });
+
+  });
+
+  describe('When challenge is already answered', () => {
+    beforeEach(async () => {
+      // given
       server.create('answer', {
         value: '2, 4',
         result: 'ko',
         assessment,
         challenge: qcmChallenge,
       });
 
+      // when
       await visit(`/assessments/${assessment.id}/challenges/${qcmChallenge.id}`);
     });
 
-    it('should mark checkboxes corresponding to the answer', async function() {
+    it('should mark checkboxes corresponding to the answer and propose to continue', async () => {
+      // then
       expect(findAll('input[type="checkbox"]')[0].checked).to.be.false;
       expect(findAll('input[type="checkbox"]')[1].checked).to.be.true;
       expect(findAll('input[type="checkbox"]')[2].checked).to.be.false;
       expect(findAll('input[type="checkbox"]')[3].checked).to.be.true;
 
       findAll('input[type=checkbox]').forEach((checkbox) => expect(checkbox.disabled).to.equal(true));
-    });
 
-  });
+      expect(find('.challenge-actions__action-continue')).to.exist;
+      expect(find('.challenge-actions__action-validate')).to.not.exist;
+      expect(find('.challenge-actions__action-validate')).to.not.exist;
```

Comment:
Doublon: il y a deux fois le même `expect`:

```suggestion
expect(find('.challenge-actions__action-validate')).to.not.exist;
expect(find('.challenge-actions__action-skip-text')).to.not.exist;
```
Est-ce cela qui était attendu ?